tests/core/secrets_collection_test.py

```diff
@@ -383,6 +387,12 @@ def test_load_baseline_without_exclude(self, mock_log):
             )
         assert mock_log.error_messages == 'Incorrectly formatted baseline!\n'
 
+    def get_point_twelve_point_eight_and_later_baseline_dict(self, gmtime):
+        # In v0.12.8 --custom-plugins got added
```

Comment:
Err, nope. This PR was made 8 months ago so I have to change any refs to 12.whatever_i_thought_this_was_gonna_be_released_in

Reply:
Okay, I have now resolved this.

```diff
diff --git a/detect_secrets/core/secrets_collection.py b/detect_secrets/core/secrets_collection.py
index b76e2f5..fafee55 100644
--- a/detect_secrets/core/secrets_collection.py
+++ b/detect_secrets/core/secrets_collection.py
@@ -115,7 +115,7 @@ class SecretsCollection:
                 # The difference will show whenever the word list changes
                 automaton, result.word_list_hash = build_automaton(result.word_list_file)
 
-        # In v0.12.8 the `--custom-plugins` option got added
+        # In v0.13.2 the `--custom-plugins` option got added
         result.custom_plugin_paths = data.get('custom_plugin_paths', ())
 
         plugins = []
diff --git a/tests/core/secrets_collection_test.py b/tests/core/secrets_collection_test.py
index 76e078f..cda37de 100644
--- a/tests/core/secrets_collection_test.py
+++ b/tests/core/secrets_collection_test.py
@@ -293,7 +293,7 @@ class TestBaselineInputOutput:
     def test_output(self, mock_gmtime):
         assert (
             self.logic.format_for_baseline_output()
-            == self.get_point_twelve_point_eight_and_later_baseline_dict(mock_gmtime)
+            == self.get_point_thirteen_point_two_and_later_baseline_dict(mock_gmtime)
         )
 
     def test_load_baseline_from_string_with_pre_point_twelve_string(self, mock_gmtime):
@@ -317,7 +317,7 @@ class TestBaselineInputOutput:
         We use load_baseline_from_string as a proxy to testing load_baseline_from_dict,
         because it's the most entry into the private function.
         """
-        original = self.get_point_twelve_to_twelve_six_later_baseline_dict(mock_gmtime)
...skipping...
-        # v0.12.8+ assertions
+        # v0.13.2+ assertions
         assert 'custom_plugin_paths' not in original
         assert secrets['custom_plugin_paths'] == ()
 
@@ -387,21 +387,21 @@ class TestBaselineInputOutput:
             )
         assert mock_log.error_messages == 'Incorrectly formatted baseline!\n'
 
-    def get_point_twelve_point_eight_and_later_baseline_dict(self, gmtime):
-        # In v0.12.8 --custom-plugins got added
+    def get_point_thirteen_point_two_and_later_baseline_dict(self, gmtime):
+        # In v0.13.2 --custom-plugins got added
         baseline = self.get_point_twelve_point_seven_and_later_baseline_dict(gmtime)
         baseline['custom_plugin_paths'] = ()
         return baseline
 
     def get_point_twelve_point_seven_and_later_baseline_dict(self, gmtime):
         # In v0.12.7 --word-list got added
-        baseline = self.get_point_twelve_to_twelve_six_later_baseline_dict(gmtime)
+        baseline = self.get_point_twelve_and_later_baseline_dict(gmtime)
         baseline['word_list'] = {}
         baseline['word_list']['file'] = 'will_be_mocked.txt'
         baseline['word_list']['hash'] = '5baa61e4c9b93f3f0682250b6cf8331b7ee68fd8'
         return baseline
 
-    def get_point_twelve_to_twelve_six_later_baseline_dict(self, gmtime):
+    def get_point_twelve_and_later_baseline_dict(self, gmtime):
         # In v0.12.0 `exclude_regex` got replaced by `exclude`
         baseline = self._get_baseline_dict(gmtime)
         baseline['exclude'] = {}
```